src/components/events/event_new/EventNewForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Form from 'react-bootstrap/Form';
+import { Button } from 'react-bootstrap';
+import './EventNewForm.css';
+import { ToggleUseDates } from './fields/ToggleUseDates';
+import { DateField } from './fields/DateField';
+import { LocationField } from './fields/LocationField';
+import { CostField } from './fields/CostField';
+import { DescriptionField } from './fields/DescriptionField';
+
+export class EventNewForm extends Component {
```

Comment:
A comment here explaining how this works would help. I understand most of how it works, but I would have to run it to really understand the result.

Reply:
Can do.